runtime/near-evm-runner/tests/standard_ops.rs

```diff
@@ -131,6 +131,7 @@ fn test_precompiles() {
 
     let mut context = create_context(&mut fake_external, &vm_config, &fees_config, accounts(1), 0);
     let (input, _) = soltest::functions::precompile_test::call();
+    println!("input: {:?}", hex::encode(&input));
```

Comment:
Perhaps this `println()` could be commented out by default? The tests should be silent by default.

Reply:
Yep